docs/tutorial/window-customization.md

```diff
@@ -114,7 +130,7 @@ const win = new BrowserWindow({
 })
```

Comment:
I think the first part of this section is already covered by the above basic tutorial